NOTICE

```diff
@@ -22,6 +22,9 @@ All contributions by Yangqing Jia:
 Copyright (c) 2015 Yangqing Jia
 All rights reserved.
 
+All contributions by Kakao Brain:
+Copyright 2019-2020 Kakao Brain
```

Comment:
I assume the legal team has reviewed these license updates and they are OK with it?

Reply:
Yes, this was added as per their guidance and I believe they mentioned they have a job that verifies licensing changes once things land. So they might reach out later if needed.